framework/assets/yii.js

```diff
@@ -329,7 +329,7 @@ yii = (function ($) {
             }
 
             if (message !== undefined) {
-                pub.confirm(message, function () {
+                $.proxy(pub.confirm, this)(message, function () {
                     pub.handleAction($this, event);
```

Comment:
Also it seems it's not the only method like that. There's `handleAction` in `else` branch. Should it be applied there as well?